webapp/src/components/SingleStudy/MapView/types.ts

```diff
@@ -0,0 +1,95 @@
+export interface NodeProperties {
+    id: string;
+    x: number;
+    y: number;
+    color: string;
+    size?: { width: number; height: number };
+    highlighted?: boolean;
+}
+
+export interface LinkSynthesis {
+    [index: string]: object;
+}
+
+export interface AreasSynthesis {
+    name: string;
+    links: LinkSynthesis;
+    thermals: string;
+    renewables: Array<string>;
+    // eslint-disable-next-line camelcase
+    filters_synthesis: Array<string>;
+    // eslint-disable-next-line camelcase
+    filters_year: Array<string>;
+}
+
+export interface AreasNameSynthesis {
+    [index: string]: AreasSynthesis;
+}
+
+export interface StudyProperties {
+    archiveInputSeries: Array<string>;
+    areas: AreasNameSynthesis;
+    bindings: Array<string>;
+    enrModelling: string;
+    outputPath: string;
+    outputs: string;
+    path: string;
+    sets: string;
+    storeNewSet: boolean;
+    studyId: string;
+    studyPath: string;
+    version: number;
+}
```

Comment:
Put this in common/types (this will be used elsewhere).
Well actually this may exist in another PR